packages/docusaurus-theme-translations/locales/zh-Hans/theme-common.json

```diff
@@ -36,7 +36,7 @@
   "theme.colorToggle.ariaLabel.mode.dark": "暗黑模式",
   "theme.colorToggle.ariaLabel.mode.light": "浅色模式",
   "theme.common.editThisPage": "编辑此页",
-  "theme.common.headingLinkTitle": "标题的直接链接",
+  "theme.common.headingLinkTitle": "标题的直接 {heading}",
```

Comment:
@Josh-Cena ?

Have no idea if I should add a space or not 🤷‍♂️

Reply:
```suggestion
  "theme.common.headingLinkTitle": "{heading}的直接链接",
```

Whether to add a space or not is a hard question. Since we should be able to insert Chinese text here, I suggest no—but if `heading` is always Latin, then my own preference is yes.